src/Funnel.sol

```diff
@@ -242,6 +277,53 @@ contract Funnel is IFunnel, NativeMetaTransaction, MetaTxContext, Initializable
         return (a.maxAmount, a.recoveryRate);
     }
 
+    /// @notice Gets the address of the base token (i.e. the underlying ERC20 token)
```

Comment:
@inheritdoc IFunnel

Reply:
This isn't in `IFunnel` and @inheritdoc doesn't work in this instance.

Reply:
Opps i meant ERC5827Proxy